cymulate-web-application-firewall/connector.py

```diff
@@ -0,0 +1,27 @@
+"""
+Copyright start
+Copyright (C) 2008 - 2024 FortinetInc.
+All rights reserved.
+FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
```

Comment:
Please check is this copyright correct

Reply:
yes, it is used in some other connectors as well